### ISSUE TITLE
broken dtb arch/arm64/boot/dts/hisilicon/hi3670.dtsi

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#373]
latest -next build is failing to build, in device tree: https://travis-ci.com/ClangBuiltLinux/continuous-integration/jobs/178750234

I assume others will spot this issue upstream.
```
arch/arm64/boot/dts/hisilicon/hi3670.dtsi:651.21-666.5: ERROR (phandle_references): /soc/ufs@ff3c0000: Reference to non-existent node or label "crg_rst"
```
cc @groeck

**Comment by nickdesaulniers** [comment]
```
arch/arm64/boot/dts/hisilicon/hi3670-hikey970.dtb
```
cc @tom-gall @ajs1984

**Comment by nathanchance** [comment]
> I assume others will spot this issue upstream.

Indeed, it was reported earlier today and the offending commit was dropped: https://marc.info/?l=linux-next&m=155061798818175&w=2

**Comment by nathanchance** [comment]
Fixed in `next-20190221`.
